gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_0/schema.yml

```diff
@@ -35,3 +35,9 @@ databaseChangeLog:
         - dropTable:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.

Reply:
I agree with graviteeio

Reply:
I added that in there because the 4.2.0 isn't released yet, but it's true that there is an alpha-1 out there... so I'll create another dedicated changelog for it